packages/donor/src/screens/HomeScreen.tsx

```diff
@@ -1,5 +1,6 @@
 import Button from "../components/basic/Button";
 import Text from "../components/basic/Text";
+import SafeScreen from "../components/basic/SafeScreen";
```

Comment:
What does this do?

Reply:
It adds padding to the page to accommodate for the notch + the lower bar on iphones , you can also disable that padding via the props if need be.

Reply:
Same as the "safe area" component, just with full width and height